Jenkinsfile

```diff
@@ -172,8 +168,21 @@ def legacyJobRunner(name) {
   }
 }
 
-def jobRunner(label, closure) {
+def jobRunner(label, useRamDisk, closure) {
   node(label) {
+    if (useRamDisk) {
+      // Move to a temporary workspace, so that we can symlink the real workspace into /dev/shm
+      def originalWorkspace = env.WORKSPACE
+      ws('/tmp/workspace') {
+        sh """
+          mkdir -p /dev/shm/workspace
+          mkdir -p '${originalWorkspace}'
```

Comment:
is the ordering correct here?  mkdir and then rmrf?

Reply:
Yeah, does the new comment help?